guides/common/modules/con_converting-a-host-to-rhel.adoc

```diff
@@ -0,0 +1,75 @@
+[id="converting-a-host-to-rhel_{context}"]
+= Converting a Host to {RHEL}
+
+You can convert {RHEL} derivative distributions into a supportable {RHEL} on a host while retaining installed applications and configurations.
+{Project} provides the Convert2RHEL utility to simplify the conversion process.
+
+The Convert2RHEL utility in {Project} consists of an Ansible role and an Ansible playbook.
+You use the Ansible role to generate conversion data on {ProjectServer}, that is, repositories, products, activation keys, and host groups.
+Then you perform the actual conversion on the host using the Ansible playbook.
+
+You can convert:
+
+* CentOS Linux 7 to {RHEL} 7
+* Oracle Linux 7 to {RHEL} 7
+* CentOS Linux 8 to {RHEL} 8
+* Oracle Linux 8 to {RHEL} 8
+
+You cannot convert a version 7 system into {RHEL} 8, or a version 8 system into {RHEL} 7.
+
+The conversion process is similar to a minor release upgrade of {RHEL} in which every RPM package on the system is replaced.
+Third-party packages and non-{Team} packages that are not available in {RHEL} are retained.
+
+The Convert2RHEL utility removes unnecessary packages such as logos, or packages known to cause issues during the conversion.
+The utility replaces the `CentOS-release` or `Oracle-release` package with the `rhel-release` package, and all packages signed by CentOS or Oracle with their {Team} equivalents.
+ifdef::satellite[]
+The utility also subscribes the host to {Team} Subscription Management.
+endif::[]
+
+The amount of time the conversion process takes can vary depending on how many packages need to be replaced, network speed, storage speed, and similar factors.
+
+.Prerequisites
+* Review https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/converting_from_an_rpm-based_linux_distribution_to_rhel/index#con_supported-conversion-paths_converting-from-a-linux-distribution-to-rhel[Supported conversion paths] in _Converting from an RPM-based Linux distribution to RHEL_.
+ifndef::orcharhino[]
```

Comment:
```suggestion
ifdef::satellite[]
```

What's the alternative for non-Satellite?

Reply:
Why can't we mention the official docs for the tool for non-Satellite?

Reply:
Oh, hang on. There are man pages in the tool that might be an alternative.

Reply:
Why can't we link to the official docs of the utility for all products? If the user uses RHEL for hosts, they need to access the official docs eventually regardless of the "Foreman" product.

Reply:
@maximiliankolb I don't understand your reasoning, can you please enlighten me on that?